packages/web-react/src/components/TextField/README.md

```diff
@@ -15,7 +16,7 @@ and show if the input is required.
 | ----------------- | ----------------------------- | ------------------------------ |
 | `id`              | string                        | Input and label identification |
 | `name`            | string                        | Input name                     |
-| `type`            | `text`, `password`            | Input type                     |
+| `type`            | `text`, `password`, `email`   | Input type                     |
```

Comment:
This would maybe deserve a standalone commit. Then it could make its way into Changelog as a new feature 🙂.

Reply:
It would, but I realized that midway through doing it. :/